src/main/java/bio/terra/cli/utils/mount/LinuxMountController.java

```diff
@@ -0,0 +1,10 @@
+package bio.terra.cli.utils.mount;
+
+import java.util.regex.Pattern;
+
+public class LinuxMountController extends MountController {
+
+  protected Pattern getMountEntryPattern() {
+    return Pattern.compile("^(\\S+)\\s+on\\s+([^\\(]+)\\s+(type[^\\)]++\\))");
```

Comment:
Can you add an explanation or example of what should/shouldn't match here? I'm having a hard time parsing this